src/Microsoft.DocAsCode.Common/FileAbstractLayer/FileWriterBase.cs

```diff
@@ -33,6 +33,18 @@ public FileWriterBase(string outputFolder)
 
         #region Help Methods
 
+        protected internal static void EnsureFolder(string folder)
+        {
+            if (string.IsNullOrEmpty(folder))
+            {
+                return;
+            }
+            if (!Directory.Exists(folder))
```

Comment:
prevent some one give folder like "c:\"

---
In reply to: [95949599](https://github.com/dotnet/docfx/pull/1172#discussion_r95949599) [](ancestors = 95949599)